blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/HttpEventsQueryHandler.java

```diff
@@ -27,35 +28,46 @@ public HttpEventsQueryHandler(EventsIO searchIO) {
 
     @Override
     public void handle(ChannelHandlerContext ctx, HttpRequest request) {
-        final String tenantId = request.getHeader("tenantId");
-        HttpResponseStatus status = HttpResponseStatus.OK;
 
-        ObjectMapper objectMapper = new ObjectMapper();
-        String responseBody = null;
-        final Timer.Context httpEventsFetchTimerContext = httpEventsFetchTimer.time();
-        try {
-            HTTPRequestWithDecodedQueryParams requestWithParams = (HTTPRequestWithDecodedQueryParams) request;
-            Map<String, List<String>> params = requestWithParams.getQueryParams();
```

Comment:
The response is being sent by DefaultHandler for the get request; There is "Tracker.trackResponse(request, response);" over there. I'm not sure if you still want this?